tests/Rules/DateBetweenTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Spatie\ValidationRules\Tests\Rules;
+
+use Carbon\Carbon;
+use Spatie\ValidationRules\Tests\TestCase;
+use Spatie\ValidationRules\Rules\DateBetween;
+
+class DateBetweenTest extends TestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+
+        Carbon::setTestNow(Carbon::create(2018, 01, 01, 00, 00, 00));
+    }
+
+    /** @test */
+    public function it_will_return_true_when_between()
+    {
+        $rule = new DateBetween(now(), now()->addDays(2));
+
+        $this->assertTrue($rule->passes('attribute', now()->addDays(1)));
```

Comment:
In a real world scenario a string would be passed as the value to a rule. Let's do that in all our tests too.